templates/default/rt/rtqueueviewbox.html

```diff
@@ -12,24 +12,13 @@
 	<TD class="nobr">
 		<span class="bold" style="{$_RT_PRIORITY_STYLES[$queue[ticket].priority]}">{$queue[ticket].id|string_format:"%06d"}</span>
 		<br>
-		{if $queue[ticket].state == $smarty.const.RT_NEW}
```

Comment:
To powoduje, że stany zgłoszeń przestają być w ogóle rozróżnialne - dużo osób nadal z tego korzysta - powiedziałbym nawet, że większość ;-)

Reply:
W sumie możemy się tego pozbyć w ogóle jak w drugim PR mamy dodanie kolorowych pól symbolizujących stan. A może stan wyróżniać po kolorze tekstu dla całego wiersza ze zgłoszeniem?

Reply:
Kolor tekstu dla całego wiersza nie robi dobrej roboty (przy >3 stanach zaczyna się festiwal kolorów). Przypominam, że prawdziwi mężczyźni odróżniają 3 kolory: fajny ped*lski wujkowy :)